doc/topics/eauth/access_control.rst

```diff
@@ -28,6 +28,13 @@ case of the peer system.
     index
     ../../ref/peer
 
+.. The two paragraphs below (in the "When to use each authentication system"
```

Comment:
Did you mean for this section to be in a `.. note::` block?

Reply:
No, that's a comment we don't want visible to the end-user. It's just telling the person who edits that particular content in the future that the content also exists in other places, and that they need to update the other instances.

Reply:
Ah, right. I don't think I've ever used an rst comment before! Thanks!